src/SonarQube.Client/Models/SonarQubeIssue.cs

```diff
@@ -55,6 +55,7 @@ public static SonarQubeIssueResolutionState ParseResolutionState(string resoluti
         {
             switch (resolution)
             {
+                case "": // TODO - check whether an empty string is a valid response
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this 'TODO' comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=csharpsquid%3AS1135)